src/env_properties.h

```diff
@@ -366,6 +366,7 @@
   V(socketaddress_constructor_template, v8::FunctionTemplate)                  \
   V(streambaseentry_ctor_template, v8::FunctionTemplate)                       \
   V(streambaseoutputstream_constructor_template, v8::ObjectTemplate)           \
+  V(noderealm_constructor_template, v8::FunctionTemplate)                    \
```

Comment:
nit: Keep the `\` lined up.

```suggestion
  V(noderealm_constructor_template, v8::FunctionTemplate)                      \
```